Casks/t/touchdesigner.rb

```diff
@@ -1,17 +1,18 @@
 cask "touchdesigner" do
   arch arm: "arm64", intel: "intel"
 
-  version "2023.11510"
-  sha256 arm:   "5a77b6d733be88b6faea42202c14d9358771c70533afe854850d52244c98a516",
-         intel: "9c83f8a245c259b70bdf379ea15b29404bbcf6c080adbeaca273289b0abc9e13"
+  version "2023.11600"
+  sha256 arm:   "5f603362a9192c60c5b3de61f00cc7d01021998935987743d9a375976a4573e4",
+         intel: "0fd39f61ccaaa0ee0d414713c7b909b211ee627f6dddc9a82019695f9e870ee1"
 
   url "https://download.derivative.ca/TouchDesigner.#{version}.#{arch}.dmg"
   name "Derivative TouchDesigner"
   desc "Tool for creating dynamic digital art"
   homepage "https://derivative.ca/"
 
   livecheck do
-    skip "No reliable way to get version info"
+    url "https://docs.derivative.ca/Release_Notes"
+    regex(/href=["'].*?["']>build\s?(\d+(?:\.\d+)+)\s?download\s?</i)
```

Comment:
```suggestion
    regex(/href=["'].*?["']>build\s?(\d+(?:\.\d+)+)\s?download\s?</i)
```

Looking back at [archive.org snapshots](https://web.archive.org/web/20240000000000*/https://docs.derivative.ca/Release_Notes) of the Release Notes page, the text around the download link has varied over the years. For example:

* Current Official Build 2023.11600 Download - Mar 04 2024 - Release Notes
* Current Build 2020.24520 - Jump to Release Notes - Download Here
* Current Build 2021.13610 - May 12 2021 - Jump to this build's Release Notes - Download Here
* The latest Official build is 2019.14650 - Download

Outside of the oldest "build is 2019.14650" format, simply matching the "Build 2023.11600" format would cover all of those variations and may be less likely to break over time. I'll push a commit to update this in a moment.